assets/js/util/test/__snapshots__/showErrorNotification.js.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Are these files needed under version control? I'm not sure what this is for.

Reply:
Yes, that's the whole point of snapshot testing.

Learn more: https://jestjs.io/docs/en/snapshot-testing